Gems/Atom/RHI/Null/Code/Source/RHI/CommandList.h

```diff
@@ -40,6 +40,7 @@ namespace AZ
             void BeginPredication([[maybe_unused]] const RHI::Buffer& buffer, [[maybe_unused]] uint64_t offset, [[maybe_unused]] RHI::PredicationOp operation) override {}
             void EndPredication() override {}
             void BuildBottomLevelAccelerationStructure([[maybe_unused]] const RHI::RayTracingBlas& rayTracingBlas) override {}
+            void UpdateBottomLevelAccelerationStructure([[maybe_unused]] const RHI::RayTracingBlas& rayTracingBlas) override {}
```

Comment:
This method also needs to be added to metal.

Reply:
I took at the RHI for Metal and I do not see any implementation details for hardware accelerated raytracing. This might be out of scope for this PR, but could be moved to future work

Reply:
You still have to add a stub like for the other RT methods, otherwise it will not compile anymore.